imports/ui/components/LoginForm.jsx

```diff
@@ -493,7 +493,8 @@ export class LoginForm extends Tracker.Component {
             id: service,
             label: capitalize(service),
             disabled: waiting,
-            type: 'link',
+            type: 'button',
```

Comment:
Though I think we need it to be a link atm, because we don't want it to submit the form and having the browser try to validate empty email fields.